src/main/java/org/commcare/formplayer/util/FormplayerDatadog.java

```diff
@@ -159,14 +145,13 @@ public void increment(String aspect, List<Tag> transientTags) {
      * @return String representing tag to send
      */
     private String getTagValueToSend(String tagName, String tagValue) {
-        // if a domain is ineligible for detailed tags, instead of sending an empty tag value, send "_other"
+        // if a domain does not have the detailed tagging feature flag enabled, instead of sending an empty tag value, send "_other"
         // this differentiates between intentionally and unintentionally empty tag values ("_other" vs "N/A", respectively)
-        String defaultValue = "_other";
         if (getDetailedTagNames().contains(tagName)) {
-            if (domain != null && getDomainsWithDetailedTagging().contains(this.domain)) {
+            if (FeatureFlagChecker.isToggleEnabled("detailed_tagging")) {
```

Comment:
what do you think about making the slug a constant with a link / ref to the HQ toggle to make cross referencing easier?

Reply:
Yeah good idea.